e2e-tests/support/tasks.js

```diff
@@ -38,6 +38,12 @@ module.exports = {
       return null;
     };
 
+    const htmlLog = (message) => {
```

Comment:
Will this be removed before the PR is merged?

Reply:
removed